Mathlib/Geometry/Manifold/Sheaf/Smooth.lean

```diff
@@ -315,12 +315,13 @@ def smoothSheafCommRing.forgetStalk (x : TopCat.of M) :
       (colimit.ι ((OpenNhds.inclusion x).op ⋙ (smoothSheafCommRing IM I M R).presheaf) U) := by
   rw [Iso.comp_inv_eq, ← smoothSheafCommRing.ι_forgetStalk_hom, CommRingCat.forget_map]
   simp_rw [Functor.comp_obj, Functor.op_obj]
+  rfl
```

Comment:
The two sides of the goal look identical, so I think this `rw` is okay.

Reply:
Do you mean the `rfl`?

Reply:
Sorry, yes!